ablySpec/RestChannel.swift

```diff
@@ -198,14 +198,40 @@ class RestChannel: QuickSpec {
                         
                         expect(message.content()).toNot(beNil())
                         expect(message.action).to(equal(ARTPresenceAction.Present))
-                        
+
                         // skip the encrypted message for now
-                        if message.payload.encoding?.rangeOfString("cipher") == nil {
+                        if message.payload.encoding.rangeOfString("cipher") == nil {
                             expect(message.content() as? NSObject).to(equal(fixtureMessage["data"].object as? NSObject))
                         }
                     }
                 }
             }
         }
+
+        // RSL4a
+        it("RSL4a") {
```

Comment:
@mattheworiordan What do you think? I need to add something to check the sent data.